### ISSUE TITLE
Replace Travis CI with Github Actions

### ISSUE THREAD

**Issue by jhedstrom** [jhedstrom/DrupalDriver#254]

**Comment by claudiu-cristea** [comment]
@jhedstrom @pfrenssen, this is fixed in the scope of #252

**Comment by AlexSkrypnyk** [comment]
@jhedstrom 
I have extracted the work @claudiu-cristea has done in #252 into a separate PR (credit preserved) into #255 

Can we please get this merged so we can unblock a bunch of PRs.
